HISTORY.md

```diff
@@ -1049,7 +761,8 @@
 
 ## 14.0.269 beta 2021-03-23
 
-* No changes made
+* chore: history: (#4760)
+* fix: support new branches in increment-version (#4761)
```

Comment:
Do we need to update these PR titles with scopes?

Reply:
I don't think so